backend/graphql/schemas/index.js

```diff
@@ -1,24 +1,28 @@
 import Job from './jobSchema'; 
+import Message from './messageSchema';
 import resolvers from '../resolvers';
 import jobResolvers from '../resolvers/jobResolvers';
 const { makeExecutableSchema } = require('graphql-tools');
 
 
 const RootQuery = `
   type RootQuery {
-      job
+      job,
+      message
   }
 `;
 
 const SchemaDefinition = `
   schema {
-    query: Job
+    query: {
+      Job,
+      Message
+    }
   }
 `;
 
-
 const schema = makeExecutableSchema({
-  typeDefs: Job,
+  typeDefs: [Job, Message],
```

Comment:
Did you did manual test? I try to do something like this and a receive  a error because I have defined two Query type. In the schema tree you have Query type in Job and Message

Reply:
Yes, you only have to put 
`extend type Query` in the others schemas in that case `Message`